pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -400,6 +400,11 @@ func (o *OpenIDCProvider) getUserInfo(ctx *context.Context, config *v32.OIDCConf
 	}
 	reusedToken, err := oauth2.ReuseTokenSource(oauth2Token, oauthConfig.TokenSource(updatedContext, oauth2Token)).Token()
 	if err != nil {
+		if strings.Contains(err.Error(), "Token is not active") ||
```

Comment:
should this error checking be externalized to some function where both the functions can use it for error checking?